.github/workflows/CI.yml

```diff
@@ -117,10 +115,3 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y clang-${{ matrix.clang-ver }} g++-multilib libgl-dev ninja-build rapidjson-dev
-
```

Comment:
Why was this removed?

Reply:
my apologies, I removed this by mistake.